### DISCUSSION TITLE
View upgraded cards

### DISCUSSION THREAD

**Discussion by Sup3rAw3som3Cod3r** [oskarrough/slaytheweb#181] [Ideas]
I think that upgrading a card should allow you to see what the upgrade does, before you choose if you want to do it or not

**Comment by oskarrough** [comment]
Definitely, that would be helpful 😅 There is a [`Campfire` room](https://github.com/oskarrough/slaytheweb/blob/main/src/ui/components/campfire.js) that handles this part in the UI.
